src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The bulldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The bulldozer stopped working");

```

Comment:
Similarly, the message in the `stopWork` method should be more informative. Consider changing it to 'Bulldozer has stopped working.' to clearly indicate which machine has stopped.